### DISCUSSION TITLE
OpenGL ES/OpenGL minimum version and other architectures

### DISCUSSION THREAD

**Discussion by realroot2185** [o3de/o3de#18330] [Q&A] [answered]
What is the OpenGL ES/OpenGL minimum version?
Can you build aarch64 and possibly other arches from a x86 pc where do you use o3de?
I have an aarch64 device with old GLES.

**Comment by jayrulez** [comment, marked as answer]
O3DE does not support OpenGL. You need Vulkan, Metal, or DX12 (or maybe WebGPU in the future).
 You can build the runtime for Android and iOS.
If by x86, you mean 32-bits, then I think you will have issues there.
To build for iOS, you need a Mac.

Building O3DE from source requires massive resources, so if you have a resource constrained device then you won't find success.

**Reply by realroot2185** [reply]
I meant x86_64.
Can you build Linux binaries for aarch64 in x86_64?
